DevVehicle35-Motors/Models/Tractor.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DevVehicle35_Motors.Interface
+{
+    internal class Tractor : iMainVehicle
```

Comment:
Interface's name should start with uppercase so: IMainVehicle. I think it's just to update your main branch on your local machine and merge it to your branch. This problem was already solved by other teams.
This will also fix your properties names that don't fit the convention. (You should be using camelcase)